README.md

```diff
@@ -5,36 +5,45 @@ This is a simple tool which allows you to vendor Helm charts from external
 sources into your repository, with the ability to apply patches to the vendored
 charts.
 
+## Requirements
+
+- Go 1.23 or newer
+- Helm 3.x installed and configured
```

Comment:
Acutally this is not needed anymore because we rely on Helm packages so it's built into chart-vendor :)